lib/org/openpsa/sales/config/config.inc

```diff
@@ -3,5 +3,7 @@
 'new_wikinote_wikiword_format' => 'note to "%s" on %s',
 'subscription_profit_months' => 12,
 'subscription_invoice_day_of_month' => false,
-
-'calculator' => 'org_openpsa_sales_calculator_default',
\ No newline at end of file
+'sales_number_format' => '#%06d',
```

Comment:
If it's only used outside the main repo, the definition should also be outside the main repo (i.e. better move it into your class)